README.md

```diff
@@ -99,6 +101,21 @@ pip3 install ffmpeg-normalize
 
 Or download this repository, then run `pip3 install .`.
 
+## Docker Build
+Download this repository and run
+
+```
+docker build -t ffmpeg-normalize .
+```
+
+Run using Windows Powershell or Linux:
+```
+docker run  -v "$(pwd):/tmp" -it docker.io/library/ffmpeg-normalize /bin/sh
```

Comment:
What is the docker.io referring to here? The image will only be local, right? Shouldn't it just be ffmpeg-normalize?

Reply:
docker.io is the default registry, hence when you build it, it will implicity get docker.io
I will remove it for brevity and confusions sake :)